.github/workflows/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] Summary of item"
+labels: bug
```

Comment:
I think we don't need `[BUG]` in the title as the `bug` label is set

Reply:
removed